rectools/models/ease.py

```diff
@@ -51,10 +54,22 @@ class EASEModel(ModelBase[EASEModelConfig]):
     ----------
     regularization : float
         The regularization factor of the weights.
+    num_threads: Optional[int], default ``None``
+        Deprecated. Number of threads used for recommendation ranking on cpu.
+    recommend_n_threads: int, default 0
+        Number of threads to use for recommendation ranking on cpu.
+        If you want to change ranking behaviour, you can manually assign new value to model
+        `recommend_n_threads` attribute. This should be done before calling  model
+        `recommend` method.
```

Comment:
```suggestion
         If you want to change this parameter after model is initialized, 
         you can manually assign new value to model `recommend_n_threads` attribute.
```

Wdyt? If agreed, lets's change for both params

Reply:
agree. fixed